src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -82,14 +86,52 @@ def color_augmenter(image, annotation,
 
 def resize_augmenter(image, annotation,
                      output_shape):
-    
 
-    new_height = tf.cast(output_shape[0], dtype=tf.int32)
-    new_width = tf.cast(output_shape[1], dtype=tf.int32)
+    resize_method = _DEFAULT_AUG_PARAMS['resize_method']
+
+    def resize_PIL_image(image, output_shape):
+        image *= 255.
+        image = image.astype('uint8')
+        pil_img = Image.fromarray(image)
+        resize_img = pil_img.resize((output_shape[1], output_shape[0]), resample=PIL.Image.BILINEAR)
+        np_img = np.array(resize_img)
+        np_img = np_img.astype(np.float32)
+        np_img /= 255.
+        return np_img
+
+    def resize_turicretae_image(image, output_shape):
```

Comment:
*turicreate

Even then, I don't love the name of this function. But in the long-run, if this approach proves stable and accurate, we should move this resizing to the C++ side anyway. There's no point in calling the C++ resizing code from Python from C++, once we converge on the right algorithm